pages/dashboard/settings.vue

```diff
@@ -14,18 +23,69 @@ const relativeRedeemed = computed(() => {
     timestamp: formatRelative(v.timestamp, now),
   }))
 })
+
+function claimVoucher(redemptionId: any) {
```

Comment:
Why is there two?

Reply:
![image](https://github.com/np-inprove/inprove/assets/66990228/97a768c6-38b3-4167-8840-6917868ad7b5)

I believe this are not the same function instead one is a to create a confirmation message before a user officially confirms to claim the voucher.

Reply:
I believe one is claimAdminVoucher and the other is claimVoucher